src/test/python/parser/opentrackio_parser.py

```diff
@@ -52,14 +52,13 @@ def main():
         print("Usage: python3 opentrackio_parser.py --file=opentrackio_sample.json --schema=opentrackio_schema.json")
         exit (-1)
 
-    sample = OpenTrackIOProtocol(sample_text,schematext,verbose) # a "Sample" is a de-serialized JSON object containing the protocol
-    sample.parse()                # parse the actual JSON of the protocol
-    sample.import_schema()        # read the schema which governs the interpretation of the protocol
+    sample = OpenTrackIOProtocol(schematext,verbose) # a "Sample" is a de-serialized JSON object containing the protocol
+    sample.parse_json(sample_text)                # parse the actual JSON of the protocol
 
-    sample.set_translation_units("cm")              # end-user preferred units
-    sample.set_sample_time_format("sec")
-    sample.set_focus_distance_units("cm")
-    sample.set_rotation_units("deg")
+    sample.set_translation_units(TranslationUnit.CENTIMETER)              # end-user preferred units
+    sample.set_sample_time_format(TimeFormat.SECONDS)
+    sample.set_focus_distance_units(FocusDistanceUnit.MILLIMETER)
+    sample.set_rotation_units(RotationUnit.DEGREE)
```

Comment:
Unit was cm but this code changed it to mm. cm was intentional to demonstrate a non-default measurement unit.